### PR TITLE
Add guides to monitor Tyk

### PR DESCRIPTION
**Pull request by sedkis**

### CODE REVIEW

**Review by letzya — CHANGES_REQUESTED** [review]
Can you please add description and tags to `benchmarks.md` (copy them from `tyk-component-monitoring.md` and update accordingly)

File: tyk-docs/content/planning-for-production/monitoring/tyk-component-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+# Monitoring Tyk Components
+
+A common question that gets asked is how to monitor the Tyk components.
+
+
+
+## Tyk Gateway
+
+The Gateway & Redis are the only components that will have a high on demand performance requirement, that scales with your API traffic.
```
Comment (letzya) [inline tyk-docs/content/planning-for-production/monitoring/tyk-component-monitoring.md]:
```suggestion
The Gateway & Redis are the only components that will have a high on-demand performance requirement, which needs to scale with your API traffic.
```

File: tyk-docs/content/planning-for-production/monitoring/tyk-component-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+A common question that gets asked is how to monitor the Tyk components.
+
+
+
+## Tyk Gateway
+
+The Gateway & Redis are the only components that will have a high on demand performance requirement, that scales with your API traffic.  
+
+### CPU Utilisation
```
Comment (letzya) [inline tyk-docs/content/planning-for-production/monitoring/tyk-component-monitoring.md]:
```suggestion
### Tyk Gateway CPU Utilisation
```

File: tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Though the Tyk Gateway is the [most performant Gateway][0] in the market, you want to avoid over/underprovisioning the hardware based on your traffic requirements.
+
+Infrastructure sizing, benchmarks, and scaling will be explained in [this section][1].
+
+
+#### 3. API Observability
+
+API Observability is the practice of monitoring holistic health of your APIs.  
+- Are you APIs behaving as intended? 
+- are you committed to your SLOs?
```
Comment (letzya) [inline tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md]:
```suggestion
- Are you committed to your SLOs?
```

File: tyk-docs/content/planning-for-production/monitoring/tyk-component-monitoring.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+
+### Liveness Health Check
+
+Health checks are extremely important in determining the status of our Tyk Components. Depending on your setup and configuration, the statuses of the following components will be returned: **Gateway**, **Dashboard**, **Redis** and **RPC**. 
+
+The health check endpoint is an expensive operation and can throttle throughput so it's important to find an optimal plan if you want to take advantage of this endpoint. The endpoint refreshes every 10 seconds and does not return statuses on the primary database (MongoDB or PostgreSQL) and the Tyk Pump component.
+
+
+```
```
Comment (letzya) [inline tyk-docs/content/planning-for-production/monitoring/tyk-component-monitoring.md]:
```suggestion
```console
```

File: tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+# What is API Infrastructure Monitoring?
+
+Infrastructure monitoring is the process of tracking and collecting system health, error counts and types, hardware resource data from our IT infrastructure (servers, virtual machines, containers, databases and other backend components) and other processes. 
+
+The infrastructure and engineering team can take advantage of real-time quantitative data by using monitoring tools to help identify trends, set alerts when a system breaks, determine the root cause of the problem and mitigate the issue. 
+
+The two main questions that your monitoring system should address: _what’s broken (symptom)_, and _why (cause)_? Successful monitoring and alerting systems should identify areas to scale, backend issues that impact users, and drive value across the organisation to improve business performance.
+
+
+### Why Is It Important?
```
Comment (letzya) [inline tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md]: I don't recall having a standard to capitalize every word in the title, or am I wrong?

File: tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+Monitoring encompasses the deep insight into the health of individual components.  Monitoring encompasses health statistics and instrumentations of servers as well as the software itself, APIs and Tyk components.
+
+Please [go here](/docs/planning-for-production/monitoring/tyk-components/) to read more about Tyk component monitoring and how to set it up.
+
+
+#### 2. Infrastructure sizing & scaling
+
+Though the Tyk Gateway is the [most performant Gateway][0] in the market, you want to avoid over/underprovisioning the hardware based on your traffic requirements.
+
+Infrastructure sizing, benchmarks, and scaling will be explained in [this section][1].
```
Comment (letzya) [inline tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md]:
```suggestion
Infrastructure sizing, benchmarks, and scaling will be explained in [this section](({{< ref "/content/planning-for-production/benchmarks/ >}}).
```

File: tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+API Observability is the practice of monitoring holistic health of your APIs.  
+- Are you APIs behaving as intended? 
+- are you committed to your SLOs?
+- Which metrics to monitor, how to retrieve them, which tools to use for dashboarding and when to alert?
+
+Please read about this in more detail [here][2].
+
+
+[0]: https://tyk.io/performance-benchmarks/
+[1]: /docs/planning-for-production/benchmarks/
```
Comment (letzya) [inline tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md]:
```suggestion
```

File: tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md (hunk 6 above)
Comment (letzya) [reply tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md]: Let netlify build this link

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md (hunk 5 above)
Comment (sedkis) [reply tyk-docs/content/planning-for-production/monitoring/tyk-monitoring.md]: Done

**Review by letzya — APPROVED** [review]

**Pull request merged**